### ISSUE TITLE
Conduct home page prototype research and synthesize findings

### ISSUE THREAD

**Issue by lauraponce** [usdoj-crt/beta-ada#45]
Conduct the research sessions with users and synthesize findings to talk through with the team. After talking through, decide together which things to focus on next and where to iterate.

Dependent on #44 

### This task is done when:

- [ ] Research sessions are completed
- [ ] Research has been synthesized 
- [ ] Shareback session has been scheduled
- [ ] Findings are written up and shared back with the team for discussion
- [ ] New issues and future work has been added to GitHub
